pkg/ruler/api_test.go

```diff
@@ -592,7 +592,7 @@ func TestRuler_PrometheusRules(t *testing.T) {
 						&alertingRule{
 							Name:   "UP_ALERT",
 							Query:  "up < 1",
-							State:  "inactive",
+							State:  "unknown",
```

Comment:
i don't know if we shouldn't change the test to let the alert evaluate for some time and end up in the inactive state. now i feel like we're changing what the test asserts. If my understanding of "unknown" is correct, now isn't the test asserting that an alert IS NOT evaluated after being loaded.

Reply:
I don't think this test asserts evaluation, but querying and filtering of rules. But I agree that this change seems like a lot, so I added another tests which ensures just that: `unknown` is the default state, and after the rule was evaluate, the state moves to `inactive`.